server/liveblog/blogs/blogs.py

```diff
@@ -256,6 +233,8 @@ def on_updated(self, updates, original):
         notify_members(blog, app.config['CLIENT_URL'], recipients)
 
     def on_delete(self, doc):
+        from .tasks import delete_blog_embed_on_s3
```

Comment:
I think it's cleaner to put imports at the top of the file. This does at least appear to be the preferred coding style in the existing code.